modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -143,7 +143,7 @@ spec:
       timeSource                        0xA0
     phc2sysOpts:                        "-a -r -n 24" <10>
     ptpSchedulingPolicy:                SCHED_OTHER   <11>
```

Comment:
There's no reason to specify ptpSchedulingPriority if it's SCHED_OTHER.  Is there a way you can indicate this is optional?

Reply:
@josephdrichard Hey! Checked callout <12> that is next to `ptpSchedulingPriority` and it says "Integer value from 1-65 used to set FIFO priority for ptp4l and phc2sys processes. *Required if SCHED_FIFO is set for ptpSchedulingPolicy.*"

Would you like me to change the `ptpSchedulingPolicy` to `SCHED_FIFO` so the set `ptpSchedulingPriority` makes sense?

I didn't put the preview links in my first comment, but here it is now: https://docs.openshift.com/container-platform/4.10/networking/using-ptp.html#configuring-linuxptp-services-as-boundary-clock_using-ptp

Reply:
I don't think that's the right preview link.
In my opinion that should say:
_Required if SCHED_FIFO is set for ptpSchedulingPolicy.  Unused Otherwise._

Reply:
This should be correct one: https://51637--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp.html#configuring-linuxptp-services-as-boundary-clock_using-ptp

Added a comment that the value is unused otherwise.